app/helpers/foreman_ansible/hosts_helper_extensions.rb

```diff
@@ -10,7 +10,7 @@ def multiple_actions
         actions = super
         if User.current.can?(:create_job_invocations) &&
            User.current.can?(:play_roles_on_host)
-          actions += [[_('Play Ansible roles'),
+          actions += [[_('Play all Ansible roles from hostgroup'),
```

Comment:
s/hostgroup/host group/ :-) it's wrong model name, let's not propagate that mistake to human language :-)

I'd also not even mention host group here, roles can be assigned to host directly. This run will include all roles from host group, its parent, its parent parent, ... and roles assigned to the host directly

we also have other place in host detail, which already says "Run Ansible roles", perhaps that would be better than Play? So I'd vote for either "Run all Ansible roles" or simply "Run Ansible roles"

And another places where this shoud probably changed are
* https://github.com/theforeman/foreman_ansible/blob/master/app/controllers/foreman_ansible/api/v2/hostgroups_controller_extensions.rb#L20
https://github.com/theforeman/foreman_ansible/blob/master/app/controllers/foreman_ansible/api/v2/hostgroups_controller_extensions.rb#L30
* https://github.com/theforeman/foreman_ansible/blob/master/app/controllers/foreman_ansible/api/v2/hosts_controller_extensions.rb#L26
* https://github.com/theforeman/foreman_ansible/blob/master/app/controllers/foreman_ansible/api/v2/hosts_controller_extensions.rb#L35
* https://github.com/theforeman/foreman_ansible/blob/master/app/views/foreman_ansible/ansible_roles/_hostgroup_ansible_roles_button.erb#L3
* https://github.com/theforeman/foreman_ansible/blob/master/app/views/foreman_ansible/ansible_roles/_hostgroup_ansible_roles_button.erb#L5

Reply:
I think it will be better to use "Run all Ansible roles" rather than "Run Ansible roles" beacuse it removes question "What Ansible roles?"

Reply:
well there can still raise a question what `all` really means, but I would agree it's a bit more suggestive :)